docs/configuration/git-providers/bitbucket-enterprise.md

```diff
@@ -1,15 +1,24 @@
-# Bitbucket Stash
+# Bitbucket Server
 
 Set your configuration values for your Bitbucket instance on the `values.yaml` file.
+
+NOTE: Since bitbucket server uses OAuth1, you'll need to create a key pair to sign and validate the requests between codacy and the bitbucket server instance:
```

Comment:
```suggestion
**NOTE:** Since Bitbucket Server uses OAuth1, you'll need to create a key pair to sign and validate the requests between Codacy and the Bitbucket Server instance:
```

Reply:
Mostly just capitalizing the names of the products. :nerd_face: